src/index.js

```diff
@@ -1,3 +1,3 @@
 'use strict';
 
-module.exports = require('./feedforwardNeuralNetwork');
+module.exports = require('./FeedForwardNeuralNetwork');
```

Comment:
instead of re-exporting FeedForwardNeuralNetwork from here, you can remove this file and change the `main` field in package.json